apps/mobile/src/libs/components/wheel/libs/components/animated-appearing-sector/animated-appearing-sector.tsx

```diff
@@ -0,0 +1,95 @@
+import { Animated, Circle } from "~/libs/components/components";
+
+import { useWheelAnimation } from "../../hooks/hooks";
+
+type Properties = {
+	animationRepetitions: number;
+	animationTime: number;
+	centerGap: number;
+	centerPoint: number;
+	endPercentInner: number;
+	endPercentOuter: number;
+	height: number;
+	layerOffset: number;
+	outlineColor: string;
+	sectorColor: string;
+	startPercentInner: number;
+	startPercentOuter: number;
+};
+
+const INITIAL_ANIMATED_VALUE = 0;
+const AnimatedCircle = Animated.createAnimatedComponent(Circle);
+
+const AnimatedAppearingSector: React.FC<Properties> = ({
+	animationRepetitions,
+	animationTime,
+	centerGap,
+	centerPoint,
+	endPercentInner,
+	endPercentOuter,
+	height,
+	layerOffset,
+	outlineColor,
+	sectorColor,
+	startPercentInner,
+	startPercentOuter,
+}: Properties) => {
+	const animatedInnerArrayDash = INITIAL_ANIMATED_VALUE;
+	const animatedInnerArrayGap = INITIAL_ANIMATED_VALUE;
+	const animatedInnerDashOffset = INITIAL_ANIMATED_VALUE;
+	const animatedInnerStrokeWidth = INITIAL_ANIMATED_VALUE;
+
+	const animatedOuterArrayDash = INITIAL_ANIMATED_VALUE;
+	const animatedOuterArrayGap = INITIAL_ANIMATED_VALUE;
+	const animatedOuterDashOffset = INITIAL_ANIMATED_VALUE;
+	const animatedOuterStrokeWidth = INITIAL_ANIMATED_VALUE;
+
+	const animatedRadius = INITIAL_ANIMATED_VALUE;
```

Comment:
Let's group these to an object here:
```suggestion
  const initialSharedValues = {
      animatedInnerArrayDash: INITIAL_ANIMATED_VALUE,
      animatedInnerArrayGap: INITIAL_ANIMATED_VALUE,
	    animatedInnerDashOffset: INITIAL_ANIMATED_VALUE,
	    animatedInnerStrokeWidth: INITIAL_ANIMATED_VALUE,
	    animatedOuterArrayDash: INITIAL_ANIMATED_VALUE,
	    animatedOuterArrayGap: INITIAL_ANIMATED_VALUE,
	    animatedOuterDashOffset: INITIAL_ANIMATED_VALUE,
	    animatedOuterStrokeWidth: INITIAL_ANIMATED_VALUE,
	    animatedRadius: INITIAL_ANIMATED_VALUE,
  }
```